stl/inc/system_error

```diff
@@ -579,7 +579,8 @@ public:
             constexpr size_t _Unknown_error_length = sizeof(_Unknown_error) - 1; // TRANSITION, DevCom-906503
             return string{_Unknown_error, _Unknown_error_length};
         } else {
-            return string{_Msg._Str, _Msg._Length};
+            return string{_Msg._Str,
+                _Msg._Length}; // lgtm [cpp/uninitializedptrfield] _Msg._Length != 0 \implies _Msg._Str is initialized

```

Comment:
```suggestion
                _Msg._Length}; // lgtm [cpp/uninitializedptrfield] (_Msg._Length != 0) == (_Msg._Str != nullptr)
```
Feel free to `_STL_INTERNAL_CHECK` the precondition and remove it from the suppression comment if you like.